scripts/modules/b2b-account/quotes.js

```diff
@@ -75,16 +80,79 @@ define([
                     });
                 }
             }
+            if (!self.model.get("b2bAccounts")) {
```

Comment:
I don't think this if should be a change done as part of this PR. I believe it's a bad rebase from the base story COM 2597 (see PR #272). It also doesn't make sense to always load B2B account data when we only need it on one of two pages.

Reply:
Removed this code

Reply:
done removed the code